src/generate_leaderboards.py

```diff
@@ -584,6 +554,9 @@ def generate_dataframe(
                 if not key.endswith("_version")
                 or key.replace("_version", "") in category_to_datasets[category]
             }
+            if "speed" not in category_to_datasets[category]:
+                logger.info(f"Model {model_id!r} is missing speed scores.")
+                metadata["speed"] = float("nan")
```

Comment:
Could you remove the speed scores from the leaderboards in this PR as well?

Reply:
Done